server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +192,31 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/permission
+     *
+     * <p>Updates a permission of a token of the specified ID.
+     */
+    @Patch("/tokens/{appId}/level")
+    @Consumes("application/json-patch+json")
```

Comment:
`json-patch+json` seems not used for this API

Reply:
Okay, I remove the annotation.